spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -41,6 +41,11 @@ It converts new messages to an internal Spring
 https://docs.spring.io/spring-integration/reference/html/messaging-construction-chapter.html#message[`Message`]
 and then sends it to the bound output channel.
 
+Google Pub/Sub treats message payloads as byte arrays. So, by default, the inbound channel adapter
+will construct the Spring `Message` with `byte[]` as the payload. However, you can also specify
```

Comment:
What happened with the rules *Sentence per line* : https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line ?